includes/Media.php

```diff
@@ -202,9 +212,9 @@ public static function rest_api_init() {
 	public static function wp_prepare_attachment_for_js( $response, $attachment ) {
 
 		if ( 'video' === $response['type'] ) {
-			$id    = get_post_thumbnail_id( $attachment );
+			$id    = (int) get_post_thumbnail_id( $attachment );
 			$image = '';
-			if ( $id ) {
```

Comment:
Why is this call to needed here `has_post_thumbnail`

Reply:
Not needed, but felt more consistent with other places where we check for the thumbnail. Could also do `0 !== $id`

Reply:
It is unnecessary here and called `get_post_thumbnail_id` instead the function which is basically the same thing.

Reply:
Clearer version of this might be.
```
             $id = 0;
			$image = '';
			if ( has_post_thumbnail( $attachment ) ) {
                $id    = get_post_thumbnail_id( $attachment );
				$image = wp_get_attachment_image_url( $id, 'medium' );
			}
			$response['featured_media']     = $id;
			$response['featured_media_src'] = $image;
```